build/calico-cni-plugin/calico-cni-plugin.spec.tmpl

```diff
@@ -0,0 +1,53 @@
+%global provider        github
+%global provider_tld    com
+%global project         projectcalico
+%global repo            cni-plugin
+%global provider_prefix %{provider}.%{provider_tld}/%{project}/%{repo}
+
+%ifarch x86_64
+%global built_arch              amd64
+%global calico_sha256           5736d46dc8f3d17eafd1d7874c49457244b698d5fccf726066c60c136ac5a318
+%global calico_ipam_sha256      ae32622d5c904e216cd914031e560e481c9ea9c213153172d0eacabadbb5e984
+%endif
+
+Name:           calico-cni-plugin
+Version:        %CALICO_VERSION%
```

Comment:
I prefer to have this hard-coded. One needs to update the spec to get a different RPM.